chart/templates/deployment.yaml

```diff
@@ -76,6 +76,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: SCRAPE_INTERVAL
+              value: '{{ .Values.prometheus.scrapeInterval }}'
```

Comment:
Please add something like `{{ .Values.(prometheus?).(scrapeInterval?) }} | default: 5` to reflect that the value is optional/missing (e.g., when upgrading from an older version of prometheus-service)

Reply:
Done!